SPECS/nbdkit/nbdkit.spec

```diff
@@ -0,0 +1,2116 @@
+%{_mandir}/man1/nbdkit-ddrescue-filter.1*
+%{_mandir}/man1/nbdkit-delay-filter.1*
+%{_mandir}/man1/nbdkit-error-filter.1*
+%{_mandir}/man1/nbdkit-exitlast-filter.1*
+%{_mandir}/man1/nbdkit-exitwhen-filter.1*
+%{_mandir}/man1/nbdkit-exportname-filter.1*
+%{_mandir}/man1/nbdkit-extentlist-filter.1*
+%{_mandir}/man1/nbdkit-fua-filter.1*
+%{_mandir}/man1/nbdkit-ip-filter.1*
+%{_mandir}/man1/nbdkit-limit-filter.1*
+%{_mandir}/man1/nbdkit-log-filter.1*
+%{_mandir}/man1/nbdkit-multi-conn-filter.1*
+%{_mandir}/man1/nbdkit-nocache-filter.1*
+%{_mandir}/man1/nbdkit-noextents-filter.1*
+%{_mandir}/man1/nbdkit-nofilter-filter.1*
+%{_mandir}/man1/nbdkit-noparallel-filter.1*
+%{_mandir}/man1/nbdkit-nozero-filter.1*
+%{_mandir}/man1/nbdkit-offset-filter.1*
+%{_mandir}/man1/nbdkit-partition-filter.1*
+%{_mandir}/man1/nbdkit-pause-filter.1*
+%{_mandir}/man1/nbdkit-protect-filter.1*
+%{_mandir}/man1/nbdkit-rate-filter.1*
+%{_mandir}/man1/nbdkit-readahead-filter.1*
+%{_mandir}/man1/nbdkit-retry-filter.1*
+%{_mandir}/man1/nbdkit-retry-request-filter.1*
+%{_mandir}/man1/nbdkit-stats-filter.1*
+%{_mandir}/man1/nbdkit-swab-filter.1*
+%{_mandir}/man1/nbdkit-tls-fallback-filter.1*
+%{_mandir}/man1/nbdkit-truncate-filter.1*
+
+%files ext2-filter
+%doc README
+%license LICENSE
+%{_libdir}/%{name}/filters/nbdkit-ext2-filter.so
+%{_mandir}/man1/nbdkit-ext2-filter.1*
+
+%files gzip-filter
+%doc README
+%license LICENSE
+%{_libdir}/%{name}/filters/nbdkit-gzip-filter.so
+%{_mandir}/man1/nbdkit-gzip-filter.1*
+
+%files tar-filter
+%doc README
+%license LICENSE
+%{_libdir}/%{name}/filters/nbdkit-tar-filter.so
+%{_mandir}/man1/nbdkit-tar-filter.1*
+
+%files xz-filter
+%doc README
+%license LICENSE
+%{_libdir}/%{name}/filters/nbdkit-xz-filter.so
+%{_mandir}/man1/nbdkit-xz-filter.1*
+
+%files devel
+%doc BENCHMARKING OTHER_PLUGINS README SECURITY TODO
+%license LICENSE
+# Include the source of the example plugins in the documentation.
+%doc plugins/example*/*.c
+%doc plugins/example4/nbdkit-example4-plugin
+%doc plugins/lua/example.lua
+%if 0%{?have_ocaml}
+%doc plugins/ocaml/example.ml
+%endif
+%doc plugins/perl/example.pl
+%doc plugins/python/examples/*.py
+#%doc plugins/ruby/example.rb
+%doc plugins/sh/example.sh
+%doc plugins/tcl/example.tcl
+%{_includedir}/nbdkit-common.h
+%{_includedir}/nbdkit-filter.h
+%{_includedir}/nbdkit-plugin.h
+%{_includedir}/nbdkit-version.h
+%{_includedir}/nbd-protocol.h
+%{_mandir}/man3/nbdkit-filter.3*
+%{_mandir}/man3/nbdkit-plugin.3*
+%{_mandir}/man1/nbdkit-release-notes-1.*.1*
+%{_libdir}/pkgconfig/nbdkit.pc
+
+%if !0%{?exclude_pkg}
+%files srpm-macros
+%license LICENSE
+%{_rpmconfigdir}/fileattrs/nbdkit.attr
+%{_rpmconfigdir}/nbdkit-find-provides
+%endif
+
+%files bash-completion
+%license LICENSE
+%dir %{_datadir}/bash-completion/completions
+%{_datadir}/bash-completion/completions/nbdkit
+
+%changelog
+* Mon Oct 31 2022 Ameya Usgaonkar <ausgaonkar@microsoft.com> - 1.30.10-3
+- Initial CBL-Mariner import from Fedora 32 (license: MIT).
```

Comment:
Not required for this PR, just a hint for the future: use latest Fedora versions, so we have a better chance of covering more CVEs.

Reply:
Without the above, spec-lint isn't happy.  Hence, I have retained it.